src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private float topSide;
+    private float downSide;
+    private float height;
+
+    public IsoscelesTrapezoid(String color, float topSide, float downSide, float height) {
+        super(color);
+        this.topSide = topSide;
+        this.downSide = downSide;
+        this.height = height;
+    }
+
+    @Override
+    public float getArea() {
+        return (topSide + downSide) / 2 * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: "
```

Comment:
let's reduce the number of lines in the draw() method
the same with other classes